mlserver-huggingface/rockcraft.yaml

```diff
@@ -43,10 +43,10 @@ parts:
       ./hack/build-wheels.sh ${CRAFT_PART_INSTALL}/opt/mlserver/dist
 
     override-stage: |
-      export ROCK_RUNTIME="huggingface"
+      export rock_RUNTIME="huggingface"
```

Comment:
I dont think we should change the env variables. same comment for changes bellow :)

Reply:
Whoops, the script replaced this :P